e2e/app/component/navigation.ts

```diff
@@ -37,8 +37,8 @@ export default class Navigation {
       await Navigation.openNavMenu(page);
       const angleIconXpath = buildXPath({ type: ElementType.Icon, iconShape: 'angle' });
       await page.waitForXPath(angleIconXpath, { timeout: 2000, visible: true });
-      const appLinkXpath = `//*[@role="button" and @tabindex="0"]//span[contains(., "${destinationApp}")]`;
-      // Find link. If not found, determine if link should be found in a submenu: User or Admin
+      const appLinkXpath = `//*[@role="button" and @tabindex="0"]//span[(text()="${destinationApp}")]`;
```

Comment:
@aweng98  I changed this xpath because 
`//span[contains(., "")]`selects the first workspaces option **YOUR WORKSPACES** , whereas `//span[(text()="${destinationApp}")] `selects the exact match.